rm-datasource/src/main/java/io/seata/rm/datasource/AbstractConnectionProxy.java

```diff
@@ -112,14 +113,7 @@ public PreparedStatement prepareStatement(String sql) throws SQLException {
         PreparedStatement targetPreparedStatement;
         SQLRecognizer sqlRecognizer = SQLVisitorFactory.get(sql, dbType);
         if (sqlRecognizer != null && sqlRecognizer.getSQLType() == SQLType.INSERT) {
-            final boolean oracle = StringUtils.equalsIgnoreCase(JdbcConstants.ORACLE, dbType);
-            final String tableName = sqlRecognizer.getTableName();
-            TableMeta tableMeta;
-            if (oracle) {
-                tableMeta = TableMetaCacheOracle.getTableMeta(getDataSourceProxy(), tableName);
-            } else {
-                tableMeta = TableMetaCache.getTableMeta(getDataSourceProxy(), tableName);
-            }
+            TableMeta tableMeta = TableMetaCacheAdapter.getTableMeta(dbType, getDataSourceProxy(), sqlRecognizer.getTableName(), false);
```

Comment:
if sqlRecognizer.getTableName() like "test" and TableMeta contains test without quotes， will it not same tableMeta？